samples/interactive-tutorials/src/main/java/events/ImportUserEventsBigQuery.java

```diff
@@ -33,26 +34,25 @@
 import com.google.longrunning.Operation;
 import com.google.longrunning.OperationsClient;
 import java.io.IOException;
+import java.time.Instant;
+import java.util.concurrent.TimeUnit;
 
 public class ImportUserEventsBigQuery {
 
   public static void main(String[] args) throws IOException, InterruptedException {
-    // TODO(developer): Replace these variables before running the sample.
     String projectId = ServiceOptions.getDefaultProjectId();
```

Comment:
I think this comment is better left in
```suggestion
    // TODO(developer): Replace these variables before running the sample.
    String projectId = "your-project-id"
```
See https://googlecloudplatform.github.io/samples-style-guide/#run-function

Reply:
I left `ServiceOptions.getDefaultProjectId()` as it gives the same result on running samples with same setup, which is important for having consistent naming for buckets etc. The comment is returned, so the user is advised to change those variables as would be necessary for them. Is this ok, or does this solution contradict too much with the guidelines?

Reply:
I think it's preferable to specify the "your-project-id" and let the use specify which project they want to use. The `getDefaultProjectID` is fine for using in the tests, but can actually be more confusing for the uninitiated user because that "default" can come from so many different places.